inference-engine/src/transformations/src/transformations/common_optimizations/hsigmoid_fusion.cpp

```diff
@@ -201,3 +204,49 @@ ngraph::pass::HSigmoidFusionWithClamp::HSigmoidFusionWithClamp() {
     auto m = std::make_shared<ngraph::pattern::Matcher>(mul_first, matcher_name);
     register_matcher(m, callback);
 }
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::HSigmoidFusionWithClampDiv, "HSigmoidFusionWithClampDiv", 0);
+
+ngraph::pass::HSigmoidFusionWithClampDiv::HSigmoidFusionWithClampDiv() {
+    MATCHER_SCOPE(HSigmoidFusionWithClampDiv);
+    // Replaces a sub-graph (Clamp(x + 3, 0, 6) / 6) with a HSigmoid op.
+    auto input = ngraph::pattern::any_input();
+    auto add_constant = ngraph::pattern::wrap_type<ngraph::opset4::Constant>();
+    auto add = ngraph::pattern::wrap_type<ngraph::opset4::Add>({input, add_constant});
+    auto clamp = ngraph::pattern::wrap_type<ngraph::op::v0::Clamp>({add});
+    auto div_constant = ngraph::pattern::wrap_type<ngraph::opset4::Constant>();
+    auto div = ngraph::pattern::wrap_type<ngraph::opset4::Divide>({clamp, div_constant});
+
+    ngraph::matcher_pass_callback callback = [=](ngraph::pattern::Matcher &m) {
+        auto &pattern_to_output = m.get_pattern_value_map();
+        auto x_output = pattern_to_output.at(input);
+
+        auto add_const_value = std::dynamic_pointer_cast<ngraph::opset4::Constant>(pattern_to_output.at(add_constant).get_node_shared_ptr());
+        auto div_const_value = std::dynamic_pointer_cast<ngraph::opset4::Constant>(pattern_to_output.at(div_constant).get_node_shared_ptr());
```

Comment:
Check for nullptr

Reply:
it's done by has_constant_value function